pages/auction/index.js

```diff
@@ -16,7 +16,10 @@ const Auctions = () => {
       <NavBar personData={personData} />
       <Link href="/auction/new">
         <a>
-          <h2 className={styles.auctionPageNavigation}>Create New Auction</h2>
+          <h2 className={styles.auctionPageNavigation}>
+            {' '}
+            <button>Create New Auction</button>
+          </h2>
```

Comment:
We wish to get rid of the h2 😅

Reply:
Any reason why` h2` and `button` is wrapped with `a` tag? 

`a` and `button` is not required here as `Link` is handling the redirection.